ios/FluentUI/Vnext/Design Token System/ListTokens.swift

```diff
@@ -55,15 +48,12 @@ public class MSFListTokens: MSFTokensBase, ObservableObject {
     @Published public var disclosureSize: CGFloat!
     @Published public var horizontalCellPadding: CGFloat!
     @Published public var iconInterspace: CGFloat!
-    @Published public var iconSize: CGFloat!
+    @Published public var iconDefaultSize: CGFloat!
```

Comment:
I believe there's value in splitting the cell tokens into a different AP.
If we do that, we'll be able to have just iconSize as a property here and set it on a "per cell" basis.